src/components/DiscordLink.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <v-list-item :title="title" prepend-icon="mdi-link" @click="handleClick" />
+</template>
+
+<script setup>
+import { computed } from 'vue';
+import { useAuthStore } from '@/stores/auth';
+const authStore = useAuthStore();
+
+const hasDiscord = computed(() => {return authStore.identities.some(({ provider }) => provider === 'discord');});
```

Comment:
Okay so this is why we're populating identies on users. I want to ensure we don't accidentally leak those out for anyone except when you request your own user, so this will warrant an inventory of requests for users